.github/CODEOWNERS

```diff
@@ -93,6 +93,7 @@ src/applications/coronavirus-vaccination-expansion @department-of-veterans-affai
 
 # My Education Benefits Team
 src/applications/my-education-benefits @department-of-veterans-affairs/my-education-benefits
+src/applications/enrollment-verification @department-of-veterans-affairs/enrollment-verification
```

Comment:
@nathanielbuck : this should be the my-education-benefits team. We don't have a separate enrollment-verification team.

Reply:
Whoops. Just pushed a fix for this.